### ISSUE TITLE
Allow set membership character "∈" in Calc for the "in" keyword for "loop"

### ISSUE THREAD

**Issue by rlwhitcomb** [rlwhitcomb/utilities#83]
as in "loop $i ∈ 'abc' { ... }" as the symbol means "in" or "belongs to" (see https://en.wikipedia.org/wiki/Glossary_of_mathematical_symbols)

**Comment by rlwhitcomb** [comment]
Also could allow  
∅
\emptyset  or ∅. for "{ }" (empty object)

**Comment by rlwhitcomb** [comment]
&isin; = &#x2208;
&emptyset; = &#x2205;
reversed empty set = &#x29B0;

See tables of other possible values:

- https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols
- https://en.wikipedia.org/wiki/Supplemental_Mathematical_Operators
- https://en.wikipedia.org/wiki/Miscellaneous_Mathematical_Symbols-A
- https://en.wikipedia.org/wiki/Miscellaneous_Mathematical_Symbols-B
- https://en.wikipedia.org/wiki/List_of_Unicode_characters (under Enclosed Alphanumerics)

**Comment by rlwhitcomb** [comment]
Marking this as a good first issue, since these are simple grammar changes -- no code should need changing.

**Comment by rlwhitcomb** [comment]
Going to leave this open until I have a chance to look through all the other symbols and enclosed alphanumerics to see if any more should be included.

**Comment by barismutan** [comment]
Hello, can I work on this issue? Or are there any issues that you have posted that you think I could help you with?

**Comment by rlwhitcomb** [comment]
According to here: https://en.wikipedia.org/wiki/Prime_omega_function we could add the two omega functions.
`Thereby ω(n) \omega (n) (little omega) counts each distinct prime factor, whereas the related function Ω(n) \Omega (n) (big omega) counts the total number of prime factors of n, honoring their multiplicity `

**Comment by rlwhitcomb** [comment]
We could implement the https://en.wikipedia.org/wiki/Prime-counting_function for powers of 10 up to 10^27 from the table there.  Not sure if that is interesting or not. But we can also use our sieve for other (small) arbitrary values.

**Comment by rlwhitcomb** [comment]
@barismutan Yes, you could help with this: I suspect there are other mathematical operators (or aliases for some existing ones) in the Unicode character tables that we could add to the grammar. If you wanted to help, you could go through the Wikipedia articles to see if there are any other operators that would make sense to support... Thanks.

**Comment by rlwhitcomb** [comment]
I could see ↦ in use for "define" as in def a($b) ↦ { ... }
&mapsto; or & # x21A6;

**Comment by rlwhitcomb** [comment]
The Calc GUI could have a popup palette field for the Unicode chars we represent (something like Character Map in Windows, or this page https://dev.w3.org/html5/html-author/charref) where you could see an explanation of the symbol, and clicking on it would insert into the Expressions field.

**Comment by barismutan** [comment]
How would I be able to find the relevant part of the code?

**Comment by rlwhitcomb** [comment]
@barismutan The code for "Calc" is all in the info.rlwhitcomb.calc package.  The ANTLR grammar is in Calc.g4, which is the best place to start. Are you familiar with ANTLR?

**Comment by barismutan** [comment]
I have had an assignment with ANTLR, I know the very basics but I am not that good.

**Comment by rlwhitcomb** [comment]
And how about `\u2303` (up arrow) for power ("**")? Or also `\u2B61`? And `\u2191`
